src/routes/airpax/utils/movementUtil.jsx

```diff
@@ -58,18 +82,26 @@ const getArrivalTime = (journey) => {
   return journey?.arrival?.time;
 };
 
-const toFormattedDepartureDateTime = (journey) => {
+const toFormattedDepartureDateTime = (journey, dateFormat = LONG_DATE_FORMAT) => {
   if (!journey?.departure?.time) {
     return UNKNOWN_TEXT;
   }
-  return getFormattedDate(journey.departure.time, LONG_DATE_FORMAT);
+  return getFormattedDate(journey.departure.time, dateFormat);
 };
 
-const toFormattedArrivalDateTime = (journey) => {
+const toFormattedArrivalDateTime = (journey, dateFormat = LONG_DATE_FORMAT) => {
   if (!journey?.arrival?.time) {
     return UNKNOWN_TEXT;
   }
-  return getFormattedDate(journey.arrival.time, LONG_DATE_FORMAT);
+  return getFormattedDate(journey.arrival.time, dateFormat);
+};
+
+const toFormattedLocation = (location) => {
+  if (!location) {
+    return UNKNOWN_TEXT;
+  }
+  const airport = getByIataCode(location);
+  return `${airport ? airport.get('city') : UNKNOWN_TEXT}, ${airport ? airport.get('country') : UNKNOWN_TEXT}`;
```

Comment:
You're checking for `airport` twice in here. This would be cleaner:
```suggestion
  if (airport) {
    return `${airport.get('city')}, ${airport.get('country')}`;
  }
  return `${UNKNOWN}, ${UNKNOWN}`;
```